django_tasks/backends/immediate.py

```diff
@@ -40,6 +40,7 @@ def enqueue(
             id=str(uuid4()),
             status=status,
             enqueued_at=enqueued_at,
+            started_at=timezone.now(),
```

Comment:
Issue: `started_at` should be run just before the task started, not once it finished.